website/docs/provisioners/habitat.html.markdown

```diff
@@ -47,20 +47,25 @@ There are 2 configuration levels, `supervisor` and `service`.  Configuration pla
 
 ### Supervisor Arguments
 * `version (string)` - (Optional) The Habitat version to install on the remote machine.  If not specified, the latest available version is used.
-* `use_sudo (bool)` - (Optional) Use `sudo` when executing remote commands.  Required when the user specified in the `connection` block is not `root`.  (Defaults to `true`)
+* `license (string)` - (Optional) Sets the acceptance of Chef licensing: https://www.chef.io/end-user-license-agreement/
```

Comment:
Given that this is a string (versus a bool for "accept the license" -- telling users what they need or the options for setting it is useful.

Reply:
I'll update that option.

Reply:
Updated the text.